value-processor/src/org/immutables/value/processor/Immutables.generator

```diff
@@ -702,26 +702,52 @@ public [type.typeAbstract.relative] [v.names.with]([v.atNullability][v.type] val
 
   /**
    * Construct a new immutable {@code [type.name]} instance.
-[for v in type.constructorArguments]
+  [for v in type.constructorArguments]
```

Comment:
I'm not against indenting `for` statement with additional 2 spaces here, but those was "outdented" on purpose so they are stick out a bit. I use this ofter in the template: combination of increasing and decreasing indentation where it might help readability. "Fixing" it here create inconsistency with how it is made in all other [untouched] places, so improvement is questionable.

Reply:
Sorry, I shouldn't have pushed that as it is probably just a remainder of when I tried to figure everything out, I will remove it again.